ext/openssl/tests/openssl_pkcs7_verify_basic.phpt

```diff
@@ -14,6 +14,11 @@ if ($contentfile === false) {
 	die("failed to get a temporary filename!");
 }
 
+$pkcsfile = tempnam(sys_get_temp_dir(), "ssl");
```

Comment:
it's better to not use temp for test files to keep it consistent (see openssl_x509_export_to_file_basic.phpt for example of the usual naming )